qml/pages/ThreadView.qml

```diff
@@ -276,7 +276,27 @@ Page {
             } else {
                 stafftag = ""
             }
-
+            var has_polls = !!post.polls  ? post.polls.length : 0
+            var polldata = []
+            if (i!=0) {
+                // can comments have polls?
+                if (has_polls) console.warn("Detected a comment with a poll. This is currently not supported.")
+            } else {
+                // reorganize the poll data into an array of objects, so we only
+                // have to result with the JSObject->ListModel conversion once:
+                // See also: Flow { id: pollsItem } below
+                for (var pi = 0; pi<has_polls; ++pi) {
+                    var pd = { "poll": {}, "votes": {} }
+                    pd["poll"] = post.polls[pi]
+                    // polls_vote is only in the data if the user has voted already
+                    if (!!post["polls_votes"]) {
+                        if (post.polls_votes[post.polls[pi].name]) {
+                            pd["votes"] = { "list": post.polls_votes[post.polls[pi].name] }
+                        }
+                    }
```

Comment:
296: adding
else {
pd["votes"] = { "list": [] }
}
seems to prevent undefined on unvoted polls can skip the loggedin check when passing submitted_votes, see note@618